frontend/src/v4/modules/viewer/betaViewer.helpers.ts

```diff
@@ -16,5 +16,5 @@
  */
 
 export const USE_BETA_VIEWER = 'useBetaViewer';
-export const setUseBetaViewer = (useBetaViewer) => localStorage.setItem(USE_BETA_VIEWER, JSON.stringify(useBetaViewer));
-export const getUseBetaViewer = () => JSON.parse(localStorage.getItem(USE_BETA_VIEWER) || 'true');
+export const setUseBetaViewer = (useBetaViewer = true) => localStorage.setItem(USE_BETA_VIEWER, JSON.stringify(useBetaViewer));
```

Comment:
The 4GB viewer state acts weird with the reset
![image](https://github.com/3drepo/3drepo.io/assets/62481250/2125e8b2-4727-4c9b-8447-eebf17024242)
Pressing reset turns the toggle off (even though on is the default)
![image](https://github.com/3drepo/3drepo.io/assets/62481250/81598ee2-239c-47d2-9e94-5eaee77ce908)
But then upon pressing save and going back into the settings it is on again
![image](https://github.com/3drepo/3drepo.io/assets/62481250/fd0e258c-a562-475b-848b-8b46ffef859b)